lib/pbench/server/api/resources/users_api.py

```diff
@@ -237,7 +212,8 @@ def post(self):
 
         try:
             auth_token = self.auth.encode_auth_token(
-                time_delta=timedelta(**token_expiry), user_id=user.id
+                time_delta=timedelta(**{"minutes": int(self.token_expire_duration)}),
```

Comment:
I don't think there is any need or reason to use the `**` operator here -- I think the following will work just as well and is much simpler:
```
                time_delta=timedelta(minutes=int(self.token_expire_duration)),
```